assets/root-files/Code/php_cs

```diff
@@ -0,0 +1,52 @@
+<?php
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->notPath('ZendTest/Code/Generator/TestAsset')
```

Comment:
@ezimuel - i forgot to mention that the paths would need truncating. All test files for a component are directly relative to the test directory now.